server/controllers/getMembers.js

```diff
@@ -0,0 +1,41 @@
+import express from 'express';
+import PostIt from './../src/PostIt';
+import verifyToken from './../middleware/verifyToken';
+import verifyMembership from './../middleware/verifyMembership';
+
+const getMembers = express.Router();
+const postIt = new PostIt();
+
+/**
+ * Gets all the members in a group
+ */
+getMembers.get(
+  '/groups/:groupId/users',
+  verifyToken,
+  verifyMembership,
+  (request, response) => {
+    postIt.findGroup(request.params.groupId)
```

Comment:
What if this group has no member ? I can't see where you sent 'not found'. Kindly look into this

Reply:
My code returns an empty array, which I also need on the frontend